drake/addpath_drake.m

```diff
@@ -60,12 +60,12 @@
 javaaddpath(fullfile(pods_get_base_path,'share','java','drake.jar'));
 
 % OSX platform-specific: check if reverted to IPv4
-if (computer('arch') == 'maci64')
+if (strcmp(computer('arch'),'maci64'))
   ipv4_preferred = java.lang.System.getProperty('java.net.preferIPv4Stack');
   if isempty(ipv4_preferred)
     ipv4_preferred = 'false';
   end
-  if (ipv4_preferred ~= 'true')
+  if (strcmp(ipv4_preferred,'true'))
```

Comment:
Hongkai I'm on it, also want to check for >= Yosemite version

On Sat, Aug 22, 2015 at 8:14 PM, Hongkai Dai <notifications@github.com>
wrote:

> In drake/addpath_drake.m
> <https://github.com/RobotLocomotion/drake/pull/1275#discussion_r37701120>:
>
> >    ipv4_preferred = java.lang.System.getProperty('java.net.preferIPv4Stack');
> >    if isempty(ipv4_preferred)
> >      ipv4_preferred = 'false';
> >    end
> > -  if (ipv4_preferred ~= 'true')
> > +  if (strcmp(ipv4_preferred,'true'))
>
> oops, will send a pull request again, thanks @rdeits
> <https://github.com/rdeits>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/RobotLocomotion/drake/pull/1275/files#r37701120>.
>